packages/grafana-schema/src/veneer/dashboard.types.ts

```diff
@@ -1,4 +1,4 @@
-import * as raw from '../raw/dashboard/x/dashboard_types.gen';
+import * as raw from '../dashboard/x/dashboard_types.gen';
```

Comment:
yeah this is what i was talking about in DM - the use of `raw` in as a path element here is independent of the raw kind category, and something we want to keep. All good, i'll fix it up :)